### DISCUSSION TITLE
issue: React beautiful dnd with useFieldArray passing down event

### DISCUSSION THREAD

**Discussion by brandonpardede** [react-hook-form/react-hook-form#9402] [Q&A]
### Version Number

6.14.1

### Codesandbox/Expo snack

https://codesandbox.io/s/react-hook-form-usefieldarray-forked-unrnkv

### Steps to reproduce

Please note that this might not be a bug report, but rather asking how i should implement it for my use case considering this might be an inevitable case

1. Go to the sandbox link i have provided
2. Input any number on those 2 inputs to identify each of the input
3. Drag on any of the input using the "Drag me" multiple times
4. Notice that there might be a slight blinking that occurs once in a while, i believe this is caused by the rerender that happens when the prop changes (which is inevitable in this case to passing down the event)
5. Compare it with the one you previously created here: https://codesandbox.io/s/react-hook-form-usefieldarray-forked-1tzyn?file=/src/index.js where it all placed in 1 single big components. The use case basically i have large scale project that has to separate the DnD context with all of the content (Droppable, Draggable) inside it, so im not going to place all of the code on this 1 big file, thats why i separate it to smaller component to receive the event from the DnD context. 

### Expected behaviour

Blinking should not occurs. Behaviour should be same as https://codesandbox.io/s/react-hook-form-usefieldarray-forked-1tzyn?file=/src/index.js

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by leapful** [comment]
@87brandonn 

Please re-check the codesandbox url, the issue url and expected behavior url are the same. Also, could you please share video or gif to give more details about the problem ?
